src/Pond.tsx

```diff
@@ -93,6 +112,8 @@ export const Pond = ({
 
   if (pond) {
     return <pondContext.Provider value={pond}>{children}</pondContext.Provider>
+  } else if (errorStateComponent) {
+    return errorStateComponent
   } else {
     return loadComponent !== undefined ? loadComponent : <></>
```

Comment:
would it be better to show a standard error component when the error handler doesn’t return one?

Reply:
not sure, it could be, that the user already has a load component, that handles the error case.

Reply:
How so? No other code could possibly know that starting the Pond has failed, only onError will know that (the load component doesn’t get this as an argument as far as I know).

Reply:
The load component is built outside and could be manipulated with Redux, React.Context, or other crazy things crawling around out there.  

On the other hand, I think the <Pond> should not have any UX implications.
If the user decides not to return a component but handle the error somehow else, probably in the parent component or in Cordova, it should be fine for us.